js/components/Breakpoints.js

```diff
@@ -1,24 +1,40 @@
 const React = require("react");
 const dom = React.DOM;
+const { connect } = require("react-redux");
+const { getSources } = require("../queries");
 
-function Breakpoints({ breakpoints }) {
+require("./Breakpoints.css");
+require("../lib/variables.css");
 
-  function onResumeClick() {
-    console.log('click')
-    gThreadClient.resume();
-  }
+const Breakpoints = React.createClass({
+
+  render() {
+    function onResumeClick() {
+      console.log('click')
+      gThreadClient.resume();
+    }
 
-  return dom.div(
-    {},
-    dom.button({ onClick: onResumeClick }, 'resume'),
-    dom.ul(
-      null,
-      breakpoints.map(bp => dom.li(
+    function getFilenameFromSources(sources, actor) {
```

Comment:
Yeah, that makes sense. Also, we have utility functions for parsing URLs. But I'm fine merging this and tweaking it later. I'll leave it up to you whether you want to merge this now or wait for this to be moved into `queries.js`.